api/ciprovider.go

```diff
@@ -0,0 +1,55 @@
+package api
+
+import (
+	"fmt"
+
+	"github.com/airware/vili/circleci"
+	"github.com/airware/vili/config"
+	"github.com/airware/vili/environments"
+	"github.com/airware/vili/kube"
+	"github.com/airware/vili/log"
+	"github.com/airware/vili/slack"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// InitializeCiClient initializes the CI client
+func InitializeCiClient(ciProvider string) error {
+	switch ciProvider {
+	case "circleci":
+		err := circleci.Init(&circleci.Config{
+			Token: config.GetString(config.CircleciToken),
+		})
+		if err != nil {
+			return err
+		}
+	case "":
+		log.Warn("You can configure a CI tool with vili in your config file with key CI_PROVIDER")
```

Comment:
This will always print a warning, even if you don't want a CI tool configured. I think this shouldn't print anything out, but instead be documented in a README.

Reply:
updated, Thanks